Docs/ProjectSpec.md

```diff
@@ -657,6 +657,7 @@ This is a convenience used to automatically generate schemes for a target based
 - [ ] **testTargets**: **[[Test Target](#test-target)]** - a list of test targets that should be included in the scheme. These will be added to the build targets and the test entries. Each entry can either be a simple string, or a [Test Target](#test-target)
 - [ ] **gatherCoverageData**: **Bool** - a boolean that indicates if this scheme should gather coverage data. This defaults to false
 - [ ] **coverageTargets**: **[[Testable Target Reference](#testable-target-reference) - a list of targets to gather code coverage. Each entry can either be a simple string, a string using [Project Reference](#project-reference) or [Testable Target Reference](#testable-target-reference)
+- [ ] **enableGPUFrameCaptureMode**: **GPUFrameCaptureMode** - Property value set for `GPU Frame Capture`.
```

Comment:
can you include the possible values?

Reply:
Added https://github.com/yonaskolb/XcodeGen/pull/1251/commits/cf29bd17cb5da6201383e5e9059885210adff11b

Reply:
wait are these the right values, or do they have to be the corresponding numbers?

Reply:
the docs should probably mention that this is only valid for the run scheme settings

Reply:
Yeah these are the right values. The string backed enum was added here: https://github.com/tuist/xcodeproj/pull/368.